pulp_rpm/app/tasks/publishing.py

```diff
@@ -695,6 +696,11 @@ def generate_repo_metadata(
         )
         sign_results = signing_service.sign(repomd_path)
 
+        signature_file_path = sign_results["signature"]
```

Comment:
Leave an in-line comment with a link to the bug please

Reply:
@dralley done